app/models/iniciative.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+class Iniciative
+  include Mongoid::Document
+
+  field :name, type: String
+  field :classification, type: String
+  field :localization, type: String
+  field :unity, type: String
+  field :tags, type: Array
+  field :url, type: String
+  field :description, type: Hash
+  field :email, type: String
+  field :contact, type: String
+
+  validates :name, :classification, :localization, presence: true
+  validates :name, length: { within: 2..100 }
+  validates :url, url: true
+
+  validate :valid_classification?, :valid_localization?, :valid_unity?, :valid_description?
+
+  def valid_description?
+    is_valid = !description.nil? &&
+               description.is_a?(Hash) &&
+               description.key?(:long)
+
+    errors.add(:description) unless is_valid
+  end
```

Comment:
Se a gente só usa a descrição longa, pra que armazenar a curta também? Em todo esse tempo que eu to no Hub a gente nunca usou.

Reply:
Vou trocar para não ser mais um Hash e sim uma String